examples/logical_interconnect_group_synergy.rb

```diff
@@ -89,6 +90,7 @@
   api_variant 'Synergy'
   scopes ['Scope1']
   action :remove_from_scopes
+  only_if { client[:api_version] == 300 && client[:api_version] == 500 }
```

Comment:
it should be || instead of &&. I am also making that change in my examples. Please change this wherever necessary

Reply:
Done